FlowServiceSample/build.gradle

```diff
@@ -2,13 +2,17 @@ apply plugin: 'com.android.application'
 
 ext.applicationId = "com.aevi.sdk.pos.flow.flowservicesample"
 ext.applicationName = "FlowServiceSample"
-ext.applicationTargetSdkVersion = 28
-ext.applicationMinSdkVersion = 21
 
 apply from: gradleScript('android/artifacts.gradle')
 apply from: gradleScript('android/versioning.gradle')
 apply from: gradleScript('android/basic-android.gradle')
 
+configurations {
+    all {
+        exclude group: 'androidx.lifecycle', module: 'lifecycle-viewmodel-ktx'
```

Comment:
Why?

Reply:
this is regarding duplicate classes found in different libraries so we need to exclude them and use only one

Reply:
```
Duplicate class androidx.lifecycle.ViewTreeViewModelKt found in modules jetified-lifecycle-viewmodel-ktx-2.3.1-runtime (androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1) and lifecycle-viewmodel-2.5.0-runtime (androidx.lifecycle:lifecycle-viewmodel:2.5.0)
```